extensions/amp-sidebar/1.0/test/validator-amp-sidebar-autoscroll-invalid.html

```diff
@@ -0,0 +1,39 @@
+<!--
```

Comment:
For now, let's just remove all validator files. The intended validation structure for our new version will be different anyway.